ui/apps/platform/src/Containers/NetworkGraph/simulation/NetworkPolicySimulatorSidePanel.tsx

```diff
@@ -14,13 +14,20 @@ import {
 } from '@patternfly/react-core';
 
 import useTabs from 'hooks/patternfly/useTabs';
+import ViewActiveYamls from './ViewActiveYAMLs';
```

Comment:
nit: keep casing of file and component consistent

It doesn't really matter whether they are both `ViewActiveYamls` or `ViewActiveYAMLS`.